components/calendar/test/calendar.visual-diff.js

```diff
@@ -0,0 +1,302 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-calendar', () => {
+
+	const visualDiff = new VisualDiff('calendar', __dirname);
+
+	let browser, page;
+
+	const firstCalendarOfPage = '#contains-today-diff-selected';
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+		await page.setViewport({width: 400, height: 1600, deviceScaleFactor: 2});
+		await page.goto(`${visualDiff.getBaseUrl()}/components/calendar/test/calendar.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	after(() => browser.close());
+
+	it('has no selected-value when selected-value not set', async function() {
+		const rect = await visualDiff.getRect(page, '#no-selected');
+		await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+	});
+
+	it('month with first row entirely current month days and last row containing next month days', async function() {
+		const rect = await visualDiff.getRect(page, '#dec-2019');
+		await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+	});
+
+	describe('localization', () => {
+
+		after(async() => {
+			await page.evaluate(() => document.querySelector('html').setAttribute('lang', 'en'));
+		});
+
+		[
+			'ar',
+			'da',
+			'de',
+			'en',
+			'es',
+			'fr',
+			'ja',
+			'ko',
+			'nl',
+			'pt',
+			'sv',
+			'tr',
+			'zh',
+			'zh-tw'
```

Comment:
The `common.js` in Intl exports an array of all the languages that you could use here instead.

Reply:
I don't think this will work because of import not working in visual-diff